src/views/announcement/show.slang

```diff
@@ -4,14 +4,7 @@
 
 nav.navigation.post-navigation role="navigation"
   div.nav-links
-    - if newer
-      div.nav-next
-        a href="/announcements/#{newer.id}"
-          span.meta-nav Previous
-          span.post-title = newer.title
-
-    - if older
-      div.nav-previous
-        a href="/announcements/#{older.id}"
-          span.meta-nav Next
-          span.post-title = older.title
```

Comment:
We may need to remove [those](https://github.com/crystal-community/crystal-ann/blob/d9c6f8f9eb968b6c27f476f13d39829542c64e4a/src/models/announcement.cr#L73-L79) unneeded methods from `Announcement` model.